packages/nextra-theme-docs/src/constants.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint sort-keys: error */
-import { useRouter } from 'next/router'
```

Comment:
should use `useRouter` from `nextra/hooks`, am I correct?

Reply:
I think yes, good catch, btw I removed this hook in v4